app/views/positions/saved_applications.haml

```diff
@@ -0,0 +1,18 @@
+%table.data_table.display#main_data_table
+  %thead
+    %tr
+      %th Date Submitted
+      %th Name
+      %th Note
+      %th Date To Review
```

Comment:
Quite right. I like capitalization of titles. You do not. But neither of us like capitalizations of prepositions.